public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,30 @@
+// Scripts for firebase and firebase messaging
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-app.js');
+importScripts('https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js');

```

Comment:
Codacy has a fix for the issue: [Replace `'https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js');` with `"https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js")`](https://app.codacy.com/gh/albertnahas/Lingwars/pullRequest?prid=9567702)
```suggestion
importScripts("https://www.gstatic.com/firebasejs/8.2.0/firebase-messaging.js")
```